firmware/shared/controls/driver_interface_torque_req_test.cc

```diff
@@ -0,0 +1,30 @@
+// don't need `#pragma once` in a .cc / source file
+
+#include "driver_interface_torque_req.h"  // include the header with your function implementation
+
+#include <iostream>
+
+#include "testing.h"
+
+int main() {
+    // Write your test cases.
+
+    // should return 0
```

Comment:
For these comments, I can see what the test "should" return from the ASSERT statement. A more useful comment is explaining why it should be 0 (in this case, because `ready_to_drive` is false)